crates/cxx-qt/Cargo.toml

```diff
@@ -19,3 +19,5 @@ categories = ["api-bindings", "gui"]
 
 [dependencies]
 cxx-qt-macro.workspace = true
+cxx.workspace = true
+cxx-qt-lib.workspace = true
```

Comment:
`cxx-qt` should not depend on `cxx-qt-lib`, i also can't see why this was added ?

Reply:
move under `dev-dependencies` if it's used by the doc tests, but curiously how does it fail without?

Reply:
Ah, I think that was failing because I tried to use QString, might not be necessary now that I'm using String :thinking:

Reply:
Still necessary, but moved to dev-dependencies now.